src/DIRAC/Resources/Computing/SingularityComputingElement.py

```diff
@@ -165,50 +168,32 @@ def __findInstallBaseDir():
             return os.path.dirname(os.path.realpath(candidate))
 
     def __getInstallFlags(self, infoDict=None):
-        """Get the flags to pass to dirac-install.py inside the container.
-        Returns a string containing the command line flags.
-        """
+        """Get the flags for installing inside the container."""
+
         if not infoDict:
             infoDict = {}
 
-        instOpts = []
-
         setup = infoDict.get("DefaultSetup")
         if not setup:
             setup = list(infoDict.get("Setups"))[0]
         if not setup:
             setup = gConfig.getValue("/DIRAC/Setup", "unknown")
         setup = str(setup)
 
-        installationName = str(infoDict.get("Installation"))
-        if not installationName or installationName == "None":
-            installationName = Operations.Operations(setup=setup).getValue("Pilot/Installation", "")
-        if installationName:
-            instOpts.append("-V %s" % installationName)
+        diracProject = "DIRAC"
+
+        project = str(infoDict.get("Project"))
+        if not project or project == "None":
+            diracProject = Operations.Operations(setup=setup).getValue("Pilot/Project", "") + project
```

Comment:
It would seem that this if statement means that project is always empty or "None" at this point when you add it to the string? Do you mean the opposite condition?

Reply:
I mean that if project is not defined then I get it from the CS

Reply:
(For clarity, my concern here was about the `+ project` on the end of the line: It would seem like that could potentially add "None" onto the end of the project name that's looked up?)

Reply:
Ah yes, I think you found a bug (fixed).